test/make.sh

```diff
@@ -55,6 +56,11 @@ compat_xargs() {
     compat=("--no-run-if-empty")
   fi
   xargs "${compat[@]}" "$@"
+  rval="$?"
+  if [[ -z "${NOWARN:-}" ]] && [[ "${rval}" -gt 0 ]]; then
+    echo "Warning: compat_xargs $* failed with exit code ${rval}" >&2
+  fi
+  return "${rval}"
 }
 
 # This function makes sure that the required files for
```

Comment:
`basefiles` function is [outdated](https://github.com/terraform-google-modules/terraform-google-memorystore/pull/10#discussion_r309299657)
cc: @ingwarr

Reply:
I would keep it for now because it's not main goal of the fix

Reply:
up 2 u